index.js

```diff
@@ -1,9 +1,41 @@
 var postcss = require('postcss'),
-	functionCall = require('reduce-function-call'),
+	valueParser = require('postcss-value-parser'),
 	assign = require('object-assign'),
 	glob = require('glob'),
 	path = require('path');
 
+function processArgs(nodes) {
+	if (!nodes.length) {
+		return [];
+	}
+	var i, max, lastIndex;
+	var args = [''];
+
+	for (i = 0, max = nodes.length; i < max; i += 1) {
+		lastIndex = args.length - 1;
+		if (nodes[i].type === 'div' && nodes[i].value === ',') {
+			args[lastIndex] = args[lastIndex].trim();
+			args.push('');
```

Comment:
What's the reasoning behind this? Why are you appending an empty string to the arguments, and why are you trimming the previous value when there's a divider rather than always trimming it on insert?